src/Routes.js

```diff
@@ -28,7 +29,8 @@ export const Routes = (props) => {
     return (
         <Switch>
             <InsightsRoute path={ paths.insights } component={ Insights } rootClass='insights'/>
-            <InsightsRoute path={ paths.cloudManagement } component={ CloudManagement } rootClass='cloudManagement'/>
+            <InsightsRoute path={ paths.cost } component={ Cost } rootClass='cost'/>
+            //* <InsightsRoute path={ paths.cloudManagement } component={ CloudManagement } rootClass='cloudManagement'/>
```

Comment:
```suggestion
            { /* <InsightsRoute path={ paths.cloudManagement } component={ CloudManagement } rootClass='cloudManagement'/> /* }
```

Reply:
It should be this way

Reply:
You are a patient person. I made the change.

Reply:
You still need the closing `*/ }` at the end of the line

Reply:
...ootClass='cloudManagement'/> */ } That's what I'm seeing at the end.